pkg/chains/chains.go

```diff
@@ -431,6 +455,17 @@ var (
 		Name:        "ton_localnet",
 	}
 
+	SuiLocalnet = Chain{
+		ChainId:     104,
```

Comment:
I don't know about this value, @gartnera do we have something specific setup in the localnet ?

Reply:
No I haven't gotten that far yet in https://github.com/zeta-chain/node/pull/3485